app/src/main/java/org/simple/clinic/bp/history/BloodPressureHistoryScreen.kt

```diff
@@ -4,13 +4,20 @@ import android.content.Context
 import android.util.AttributeSet
```

Comment:
Reword the commit message to be more descriptive?

Ex: `Forward adapter events to Mobius loop`